README.md

```diff
@@ -1 +1,66 @@
-# needlehack
\ No newline at end of file
+# needlehack
+
+The project imports, normalizes and stores this information from different RSS declared in an OPML file: [engineering_blogs.opml](.src/main/resources/opml/engineering_blogs.opml)
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+- [Java8](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
+- [Maven 3.5.0](https://maven.apache.org/download.cgi)
+
+You have to install Docker and Docker Compose:
+
+- [Docker](https://docs.docker.com/installation/#installation)
+- [Docker Compose](https://docs.docker.com/compose/install)
+
+### Installing
+
+* Clone source code from git
+
+```
+$ git clone git@github.com:needlehack-team/needlehack.git
+```
+
+* Build project with Maven
+
+```
+$ mvn clean install
+```
+
+* Run the application from the command line using:
+
+```
+$ ./run.sh
+```
+
+## Built With
+
+* [Spring Framework](https://spring.io/) - The framework used
+* [Maven](https://maven.apache.org/) - Dependency Management
+* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
+* [Elasticsearch](https://www.elastic.co/products/elasticsearch) - Search engine based on [Lucene](https://lucene.apache.org/core/)
+
+## Versioning
+
+We use [SemVer](http://semver.org/) for versioning.
+
+## Authors
```

Comment:
I suggest joining **Author** and **Contributors** sections on **Team** section

Reply:
Done!